tests/performance-test/deploy/config/minimal-collectd.conf

```diff
@@ -1,23 +1,41 @@
 Interval 1
 
+LoadPlugin "log_logstash"
+<Plugin "log_logstash">
+  LogLevel "notice"
+  File "/tmp/events.json"
```

Comment:
I was wondering what this was about, and it took almost until the end of the story for it to show itself. Clever!